DigitalLearningSolutions.Web/Views/LearningPortal/Current/Current.cshtml

```diff
@@ -31,13 +31,13 @@
                                     text="@LearningHubWarningTextHelper.ResourceDetailsMayBeOutOfDate" />
     }
 
+    <partial name="SearchablePage/Configurations/_FullWidthSearchSortAndFilter" model="Model" />

```

Comment:
I moved this out of the NoDataFound if statement for two reasons:

1. The Complete and Available pages still display the controls when no data is found. This makes the pages consistent
2. There was a bug with updating the searchable links in JS (part of 621) that prevented the spinner from stopping when no search bar was present. Now, the search bar is always rendered, so the code hides the spinner properly.